docs/modules/project/pages/governance.adoc

```diff
@@ -0,0 +1,38 @@
+= Asciidoctor.js Governance
```

Comment:
Using governance confused me at first. To me, this is a board composition or a contact list, not rules defining the project's governance. That said, once I understood that Governance meant to refer to project leaders and core members I was less confused.

Reply:
I didn't know what else I should add because we don't have meetings and the organization is pretty flat. So it's really "who can do what" page :wink: 
If you think of a better title I'm all ears :ear: